lexicons/com/atproto/sync/subscribeRepos.json

```diff
@@ -118,7 +119,31 @@
       "properties": {
         "seq": { "type": "integer" },
         "did": { "type": "string", "format": "did" },
-        "time": { "type": "string", "format": "datetime" }
+        "time": { "type": "string", "format": "datetime" },
+        "handle": {
+          "type": "string",
+          "format": "handle",
+          "description": "The current handle for the account, or 'handle.invalid' if validation fails. This field is optional, might have been validated or passed-through from an upstream source. Semantics and behaviors for PDS vs Relay may evolve in the future; see atproto specs for more details."
+        }
+      }
+    },
+    "account": {
+      "type": "object",
+      "description": "Represents a change to an account's status on a host (eg, PDS or Relay). The semantics of this event are that the status is at the host which emitted the event, not necessarily that at the currently active PDS. Eg, a Relay takedown would emit a takedown with active=false, even if the PDS is still active.",
+      "required": ["seq", "did", "time", "active"],
+      "properties": {
+        "seq": { "type": "integer" },
+        "did": { "type": "string", "format": "did" },
+        "time": { "type": "string", "format": "datetime" },
+        "active": {
+          "type": "boolean",
+          "description": "Indicates that the account has a repository which can be fetched from the host that emitted this event."
+        },
+        "status": {
```

Comment:
I'm alright with keeping `active` around, though I am not sure it is necessary.  It allows a PDS to express some conflicting information, like setting a `status` with `active=true`.

Reply:
just to expand on the reasoning there a bit, I wanted to future-proof a bit against us adding additional status strings in the future. If simple infra implementations match on the status string to either make the account visible or not make the account visible (who knows what kind of weird match/switch control flow they might have), then adding a status code could result in the incorrect behavior. Making it a boolean flag makes the "is account visible or not" behavior very clear, with the status being more of an end-user-visible context and less of a high-stakes "is content redistributed at all".

still agree with your point, just wanted to document that bit of thinking